packages/react-scripts/scripts/utils/isSsr.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const fs = require('fs');
+const paths = require('../../config/paths');
+
+/**
+ * In order to build for SSR, need to confirm 3 places
+ * 
+ * 1. `ssrEnabled` is on
+ * 2. Use `build-ssr` command
+ * 3. `ssr.js` exists in `src` folder
+ * 
+ */
+const isSsr = () => {
+  // Ensure `ssrEnabled` is on
+  const appPackageJson = require(paths.appPackageJson);
+  const bpkReactScriptsConfig = appPackageJson['backpack-react-scripts'] || {};
+  const ssrEnabled = bpkReactScriptsConfig.ssrEnabled || false;
```

Comment:
> Maybe the default here should be true if the config is not provided?
> 
> This will ensure that existing users will not need to add a config variable and break for them, and users should opt-out?

The SSR projects is few, `false` default value is intentionally set here, this will make a deeper impression on engineers for they should know what they are doing is to make projects run with SSR mode.